src/EditorFeatures/CSharpTest/CompleteStatement/CSharpCompleteStatementCommandHandlerTests.cs

```diff
@@ -0,0 +1,1296 @@
+        public void ArgumentListOfMethodInvocation_MultiLine()
+        {
+            var code = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x$$, 
+    y)");
+
+            var expected = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x, 
+    y);$$");
+
+            VerifyTypingSemicolon(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void ArgumentListOfMethodInvocation_MultiLine2()
+        {
+            var code = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x$$, 
+    y");
+
+            var expected = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x, 
+    y);$$");
+
+            VerifyTypingSemicolon(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void ArgumentListOfMethodInvocation_MultiLine3_knownfailure()
+        {
+            // Not yet handling scenario where closing paren is on the next line
+            var code = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x$$, 
+    y
+    )");
+
+            var expected = CreateTestWithMethodCall(@"
+var test = ClassC.MethodM(
+    x, 
+    y
+    );$$");
+
+            VerifyTypingSemicolon(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void PropertyAccessors1()
+        {
+            var code = @"
+public class ClassC
+{
+    private int xValue = 7;
+    public int XValue
+    {
+        get
+        {
+            return Math.Min(xValue$$, 1)
+        } 
+    }
+}";
+
+            var expected = @"
+public class ClassC
+{
+    private int xValue = 7;
+    public int XValue
+    {
+        get
+        {
+            return Math.Min(xValue, 1);$$
+        } 
+    }
+}";
+
+            VerifyTypingSemicolon(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void PropertyAccessors2()
+        {
+            var code = @"
+public class ClassC
+{
+    private int xValue = 7;
+    public int XValue
+    {
+        get
+        {
+            return Math.Min(Math.Max(xValue,0$$), 1)
+        } 
+    }
+}";
+
+            var expected = @"
+public class ClassC
+{
+    private int xValue = 7;
+    public int XValue
+    {
+        get
+        {
+            return Math.Min(Math.Max(xValue,0), 1);$$
+        } 
+    }
+}";
+
+            VerifyTypingSemicolon(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void PropertyAccessors3()
+        {
+            var code = @"
+public class Person
+{
+   private string firstName;
+   private string lastName;
+   
+   public Person(string first, string last)
+   {
+      firstName = first;
+      lastName = last;
+   }
+
+   public string Name => {firstName} {lastName}$$   
```

Comment:
:confused: the expression body seems incorrect. Did you mean to wrap it in `$""`?